examples/retrieve-token/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "retrieve-token-example"
+description = "Example to retrieve a token from System Identity Service using the library"
+version = "0.0.1"
+edition = "2021"
+authors = ["The New Relic Agent Control Team"]
+publish = false
+license-file = "./../LICENSE.md"
+
+[dependencies]
+nr-auth = { path = "../../" }
+url = "2.5.4"
+http = "1.3.1"
+thiserror = "2.0.12"
+reqwest = { version = "0.12.11", default-features = false, features = [
+  "blocking",
+  "rustls-tls",
+  "rustls-tls-native-roots",
+  "socks",
+] }
+dotenvy = "0.15.7"
```

Comment:
would make sense to pin majors, or major/minor only? just for the sake of having less renovate pr's

Reply:
Sounds good to me :) 

Done in dd1bef7d31d4b5f7b2b9b2bcc5eafee7da253f0e